### PR TITLE
Build a Pulumi VS Code Devcontainer

### PR DESCRIPTION
**Pull request by spara**

### CODE REVIEW

File: content/blog/vscode-devcontainers/index.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+meta_image: devcontainer.png
+authors:
+    - sophia-parafina
+tags:
+    - "VS Code"
+    - "devcontainers"
+    - Docker
+---
+
+One of the major advantages of using containers for development is reducing the need to install software and all of the associated dependencies. Developers can start writing code without configuring a development environment that emulated production. The Visual Studio Code Remote - Containers extension lets you develop inside a container. If you want to use Pulumi’s infrastructure as code engine without having to install the Pulumi CLI, this blog post is for you!
```
Comment (jkisk) [inline content/blog/vscode-devcontainers/index.md:14]: perhaps emulates instead of emulated?

File: content/blog/vscode-devcontainers/index.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+1. Next, we’ll create the devcontainer configuration files. Click on the Remote Status button and select **Remote Container - Add Development Container Configuration Files ...**.
+
+    ![A configuration files](add_devcontainer_config_files.png)
+
+1. Select **Debian 10 & Git**. We’ll replace the Dockerfile with the Pulumi Dockerfile.
+
+    ![Select config file](select_config_file.png)
+
+    VS Code will display a popup asking to build the container. We’ll replace the Dockerfile with the Pulumi Dockerfile, dismiss the popup.
+
```
Comment (jkisk) [inline content/blog/vscode-devcontainers/index.md:61]: I might reverse the order of the last sentence "Dismiss the popup, we will replace the Dockerfile with the Pulumi Dockerfile."  I have a bad habit of clicking before I get to the end of sentences though.

File: content/blog/vscode-devcontainers/index.md (hunk 1 above)
Comment (spara) [reply content/blog/vscode-devcontainers/index.md:14]: fixed

File: content/blog/vscode-devcontainers/index.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+    ```
+
+1. Next, set your provider credentials on your machine and add the following to *devcontainer.json*.
+
+    ```json
+    "containerEnv": {
+        "AWS_ACCESS_KEY_ID": "${localEnv:AWS_ACCESS_KEY_ID}",
+    "AWS_SECRET_ACCESS_KEY": "${localEnv:AWS_SECRET_ACCESS_KEY}"
+    },
+    ```
```
Comment (praneetloke) [inline content/blog/vscode-devcontainers/index.md:116]:
```suggestion
    ```json
    "containerEnv": {
        "AWS_ACCESS_KEY_ID": "${localEnv:AWS_ACCESS_KEY_ID}",
        "AWS_SECRET_ACCESS_KEY": "${localEnv:AWS_SECRET_ACCESS_KEY}"
    },
    ```
```

**Pull request merged**